lib/ffmpeg/qsv/util.py

```diff
@@ -73,6 +73,10 @@ def mapprofile(codec, profile):
       "profile0"  : "profile0",
       "profile1"  : "profile1",
     },
+    "vp9-10" : {
+      "profile1"  : "profile1",
+      "profile2"  : "profile2",
+    },
```

Comment:
profile1?  Should be profile2 and profile3, right?

Reply:
You are right, updated